CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+## Development environment
+
+Get the latest development version. Fork and clone the repo:
+
+```cli
+$ git clone git@github.com:<your-username>/helm-charts.git
+```
+Install [Helm](https://helm.sh/docs/intro/install/) and [Skaffold](https://skaffold.dev/docs/install/).
+
+Install your local Kubernetes cluster:
+- [Docker Desktop](https://docs.docker.com/desktop/kubernetes/) - recommended
+- [Kind](https://kind.sigs.k8s.io/docs/user/configuration/) - it needs additional configuration 
+   to expose ingress 👇
+- K3d - not supported yet
+
+
+### Kind cluster
+Kind cluster does not expose ports by default. To enable ingress, you need to create cluster with
+following command:
+
+```cli
+$ kind create cluster --config kind-config.yaml
+```
+
+That's it. You should be ready to make changes, run tests, and make commits! If
+you experience any problems, don't hesitate to ping us in our [chat](/chat).
```

Comment:
Which chat? :grin: 
Link doesn't work for me

Reply:
GPT :D ... I thought I removed this ... :D